google/cloud/forseti/services/inventory/base/resources.py

```diff
@@ -428,10 +440,21 @@ def iter(self):
                 yield FACTORIES['dataset'].create_new(data)
 
 
+class ComputeIterator(ResourceIterator):
+    def iter(self):
+        gcp = self.client
+        if (self.resource.enumerable() and
+                self.resource.compute_api_enabled(gcp)):
+            data = gcp.fetch_compute_project(
+                projectid=self.resource['projectId'])
+            yield FACTORIES['compute'].create_new(data)
+
```

Comment:
if not self.resource.compute_api_enabled(gcp):
    raise error
Thanks!

Reply:
Changed based on conversation. This function now raises from fetch_compute_project(), other compute API calls check if the API is enabled first. This now logs ApiNotEnabledError for projects when the Compute API is not enabled.